frontend/App.vue

```diff
@@ -5,60 +5,64 @@
     </v-app-bar>
 
     <v-main>
-      <VehiclesList :vehicles="this.vehicles" @createVehicle="createVehicle" @deleteVehicle="deleteVehicle" />
+      <Vehicles @createVehicle="createVehicle" @deleteVehicle="deleteVehicle" :vehicles="vehicles" />
```

Comment:
```suggestion
      <vehicle-list @create="create" @delete="delete" :vehicles="vehicles" />
```
Go with the two word naming as it is written in the [style guide](https://v2.vuejs.org/v2/style-guide/?redirect=true). Also, use dash-case ☝️ 
As for the listeners and method naming, move this vehicle state management logic in the vehicles root index file and then you can simply use create, remove etc.

Reply:
this throws errors as delete is javascript keyword

Reply:
`@remove` shall do the trick then :)